vividus-tests/src/main/resources/story/replacement/ReplacedStepsExpected.story

```diff
@@ -0,0 +1,19 @@
+Scenario: Test replacing of deprecated step without parameters
```

Comment:
please add metatags, some spacings between steps, maybe examples table, just to make sure it replaces only expected things and has no side effects